apps/cdk/src/app/production-backend.ts

```diff
@@ -0,0 +1,19 @@
+import { App } from 'aws-cdk-lib';
+
+import { BackendStack } from '../stacks/backend';
+import { waitForStackIdle } from '../wait-for-stack-idle';
+
+/**
+ * Deploy the production backend.
+ */
+async function main() {
+    const stackName = 'antalmanac-backend';
```

Comment:
1. nit: I feel like the name should have prod in it somewhere
2. not a nit: The name of our current prod backend (which is very much my fault but anyways) is `prod-us-west-1-Backend` which means that there is no way this stack will deploy as it isn't going to overwrite the current stack. The main problem is that the prod user data db is in that stack so we can't just delete it and redeploy. The database will need to be extracted from the stack no matter what before it is deleted, after that we can either try and import it with Cloudformation (very not fun) or figure out how to copy paste the data from one DynamoDB to another (uncertain how not fun I have never done this).

There is a similar issue with the dev backend but that can just be deleted as the data on there isn't really important.

Reply:
Hmm: https://www.npmjs.com/package/copy-dynamodb-table :eyes:

Reply:
Gonna leave this comment open as a reminder when merging this PR